### PR TITLE
Sphinx documentation

### PR DESCRIPTION
**Pull request by FedericoBerlfein**
Adding Sphinx documentation files. This is the first version, still need to make a few checks before merging.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import sys
+sys.path.insert(0,os.path.abspath('.'))
+
+import euclidlike
+import euclidlike_imsim
+
+project = 'GalSim-Euclid-Like'
+copyright = '2024, Rachel Mandelbaum, Axel Guinot, Federico Berlfein, Andy Park, Xiangchong Li, Michael Troxel, Tianqing Zhang'
+author = 'Rachel Mandelbaum, Axel Guinot, Federico Berlfein, Andy Park, Xiangchong Li, Michael Troxel, Tianqing Zhang'
+release = '0.0.1'
```
[the remark this answers is not in the document]
Comment (FedericoBerlfein) [reply docs/conf.py]: Make `VERSION` file.

File: docs/conf.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+import os
+import sys
+sys.path.insert(0,os.path.abspath('.'))
+
+import euclidlike
+import euclidlike_imsim
+
+project = 'GalSim-Euclid-Like'
+copyright = '2024, Rachel Mandelbaum, Axel Guinot, Federico Berlfein, Andy Park, Xiangchong Li, Michael Troxel, Tianqing Zhang'
```
[the remark this answers is not in the document]
Comment (FedericoBerlfein) [reply docs/conf.py:17]: Rachel will ask once documentation is live.

File: CHANGELOG.rst [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Changes from v0.0.0 to v0.0.1
```
[the remark this answers is not in the document]
Comment (FedericoBerlfein) [reply CHANGELOG.rst:1]: Change 0.9.0

**Comment by FedericoBerlfein** [comment]
Made last couple of changes based on last weeks conversation. Mainly:

- Changed version number to 0.9.0
- Made VERSION txt file in repo root.
- Added `_version.py` file to `euclidlike_imsim`
- Added basic installation instructions in `README.rst`
- Changed "classes and functions" section to "API" in `euclidlike` and `euclidlike_imsim`

I think this PR is ready for a final review and potential merge.

**Review by aguinot — APPROVED** [review]

File: docs/examples.rst [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+=========
+
+The ``GalSim-Euclid-Like/examples`` directory contains example files for how use the euclidlike module.  
+
+End-to-end demo 
+---------------
+
+:gh-link:`end_to_end_demo.py <examples/end_to_end_demo.py>`
+
+This first demo is the euclidlike-equivalent of  _`demo 13 <https://github.com/GalSim-developers/GalSim/blob/main/examples/demo13.py>`_ in ``GalSim``. This demo uses the Euclid-like PSF, WCS, and background noise to produce a realistic scene of galaxies and stars as observed from a Euclid-like Telescope.
```
Comment (rmandelb) [inline docs/examples.rst:11]: Something is going wrong with this line, if you look at the html it produces on your branch, but I'm not sure how to fix it.

File: docs/examples.rst [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+The ``GalSim-Euclid-Like/examples`` directory contains example files for how use the euclidlike module.  
+
+End-to-end demo 
+---------------
+
+:gh-link:`end_to_end_demo.py <examples/end_to_end_demo.py>`
+
+This first demo is the euclidlike-equivalent of  _`demo 13 <https://github.com/GalSim-developers/GalSim/blob/main/examples/demo13.py>`_ in ``GalSim``. This demo uses the Euclid-like PSF, WCS, and background noise to produce a realistic scene of galaxies and stars as observed from a Euclid-like Telescope. 
+
+** Features introduced in the Python file**:
```
Comment (rmandelb) [inline docs/examples.rst:13]: These asterisks are not doing what was intended (if you look at the html it produces)

**Review by rmandelb — APPROVED** [review]
Just two small issues I noted in the documentation of the examples.  I'm hitting "approve" because you might be able to fix it easily and then you can go ahead and merge.

File: docs/examples.rst (hunk 5 above)
Comment (FedericoBerlfein) [reply docs/examples.rst]: Addressed.

File: docs/examples.rst (hunk 4 above)
Comment (FedericoBerlfein) [reply docs/examples.rst]: Addressed.

**Pull request merged**